scripts/linux/install-kubernetes.sh

```diff
@@ -13,16 +13,42 @@ while true; do
   esac
 done
 
+ansible_debug="$1"
+
+echo "Ansible debug: $ansible_debug"
+
+verbose_flag=
+if [ "$ansible_debug" = 'on' ]; then
+    verbose_flag="-vv"
+fi
+
+echo "Verbose verbose_flag: $verbose_flag"
+
 echo "Ensure the Docker service is enabled and running"
+
+exit
```

Comment:
?

Reply:
fixed (working at 3 am is not always good...) :-)